src/oscar/apps/customer/forms.py

```diff
@@ -79,21 +79,11 @@ def get_reset_url(self, site, request, user, use_https):
 
 
 class SetPasswordForm(auth_forms.SetPasswordForm):
-
-    def clean_new_password2(self):
-        new_password2 = super(SetPasswordForm, self).clean_new_password2()
-        # For backward compatibility with Django 1.8
-        validate_password(new_password2, self.user)
-        return new_password2
+    pass
```

Comment:
Why do we need those forms at all without no customization? We can just use raw Django auth forms in the views.

Reply:
For backward compatibility - projects might be importing these classes. I will add a comment marking them as deprecated.

Reply:
I have added a deprecation decorator to both classes. We can remove them in the next release.